support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.jsx

```diff
@@ -151,7 +151,8 @@ function updatePayerEmail(data: Object, setEmail: string => void) {
 }
 
 function updatePayerName(data: Object, setFirstName: string => void, setLastName: string => void): boolean {
-  const nameParts = data.payerName.split(' ');
+  // NB: This turns "    jean    claude    van    damme     " into ["jean", "claude", "van", "damme"]
+  const nameParts = data.payerName.trim().replace(/\s+/g, ' ').split(' ');
```

Comment:
The replace() regex will catch everything from the trim, but I guess there's no harm!

Reply:
so my thinking is that the trim will totally *remove* any whitespace before and after the name and the replace will go inside and replace multiple instance of whitespace with a single space

Reply:
just calling the replace could leave a single space before and after the name

Reply:
Ah yes sorry I missed the single space in the regex replace value, I thought it was empty-string. More tea required this morning.